ADR/0008-integration-service-api-contracts.md

```diff
@@ -0,0 +1,248 @@
+# 8. StoneSoup Test Stream - API contracts
+
+Date: 2023-01-[]
+
+## Status
+
+Accepted
+
+## Context
+
+The Hybrid Application Cloud Build Service being developed aims to serve Red Hat teams but also partners and customers. This requires a level of adaptability to avoid recreating custom flows and Tasks for each stakeholder.
```

Comment:
Hybrid Application Cloud Build Service == HACBS but I don't know what the new way of saying this would be. :-/

Reply:
So let's use `The Stonesoup project` now, and we may update it if it has a different name. Thanks.